packages/world/src/resourceTypes.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+bytes2 constant RESOURCE_NAMESPACE = "ns";
+bytes2 constant RESOURCE_TABLE = "tb";
+bytes2 constant RESOURCE_TABLE_OFFCHAIN = "tf";
+bytes2 constant RESOURCE_SYSTEM = "sy";
+bytes2 constant RESOURCE_MODULE = "md";
```

Comment:
I wish Solidity had something like TypeScript's enums where you can define the value of each, e.g.
```ts
enum ResourceType {
  NAMESPACE = 'ns',
  TABLE = 'tb',
  ...
}
```